Guardfile

```diff
@@ -3,7 +3,7 @@
 
 notification :growl
 
-guard :minitest, spring: true, env: {GUARD: 'true'}, all_env: :GUARD_COVERAGE, all_after_pass: true do
+guard :minitest, spring: true, env: {GUARD: 'true'}, all_env: :GUARD_COVERAGE, all_after_pass: false do
```

Comment:
i may not have intended to check that in. but yes - this particular repo is pretty slow. especially when testing through docker. makes it a pain to fix a particular test if i have to wait minute(s) to see the results.

and you can always hit enter to run all the tests again, right?